data/pages-content/tournament.md

```diff
@@ -0,0 +1,201 @@
+---
+title: EuroPython 2023 AI game tournament
+subtitle: Participate in the EuroPython 2023 AI game tournament!
+---
+
+## Register your team here https://forms.gle/v2s9oHZ1xLTUo5ov7 !
+
+![supremacy](https://github.com/nvaytet/supremacy/assets/39047984/6e800f7c-6eb4-47d4-9621-1465aeaae879)
+
+# EuroPython 2023 AI game tournament
+
+## What is the tournament about?
+
+We have created a video game that is designed to be played by a small python program, rather than a human.
+Conference attendees can participate in a tournament where they (either alone or in a team) each submit a bot that will play the game, and at the end of the conference, we will have a tournament session where everyone can come and watch our strange creations play against each other (either brilliantly or it may all go wrong!).
+
+## Disclaimer
+
+<img src="https://upload.wikimedia.org/wikipedia/commons/thumb/4/49/Flag_of_Ukraine.svg/320px-Flag_of_Ukraine.svg.png" width="50" />
+
+The topic of the game is a sci-fi war on a foreign planet, set far in the future. We understand that the topic of war could upset some people with the unfortunate events that are currently going on in Ukraine. We do not want to promote war, and we fully support Ukraine in their hardship. Development of the game was started before the invasion of Ukraine, it is just a coincidence that this is the game that was ready for the conference.
+We just want to make a fun event for EuroPython participants, we do not want to offend anyone.
```

Comment:
How about a neutral “we don't condone war and violence” disclaimer?
Sadly, there is more than one armed conflict in the world right now, so calling out one specific conflict doesn't make much sense to me. It's like the other conflicts don't matter.

Reply:
> It's like the other conflicts don't matter.

Yes, I was thinking about that also. I will change to make it shorter and not name Ukraine specifically.

Reply:
I think the disclaimer now is better!
We'll see what others say when they review the PR...

Hopefully I'll see you at the tournament :D